src/installation/guides/chroot.md

```diff
@@ -0,0 +1,222 @@
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Use xbps to boot strap the distribution by installing the `base-system`
+metapackage.
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+## Configuration
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount -t proc /proc /mnt/proc
+# mount -t sysfs /sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the DNS configuration into the new root so that xbps can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+Write the hostname to `/mnt/etc/hostname`. Go through the options in
+[`/mnt/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/mnt/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+[configure at least one super user account](../../config/users-and-groups.md).
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX` with their respective UUID, which can be found
+by running `lsblk -f` (see [lsblk(8)](https://man.voidlinux.org/lsblk.8)).
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following fstab:
+
+```
+/dev/sda1 /boot/EFI vfat rw,relatime,[...] 0 0
+/dev/sda2 / ext4 rw,relatime 0 0
+/dev/sda3 /home xfs rw,relatime 0 0
+```
+
+The fstab is modified thus with information from `lsblk -f`:
+
+```
+UUID=6914[...] /boot/EFI vfat rw,relatime,[...] 0 2
+UUID=dc1b[...] / ext4 rw,relatime 0 1
+UUID=1f4c[...] /home xfs rw,relatime 0 2
+```
```

Comment:
Do you think you could line up these entries? I think it makes it easier for people to understand what's in each column, and it clearly separates each field.

Reply:
The two entries below as well.

Reply:
And if #164 is merged, it would be cool to add a reference to it too

> if you're setting up your system with an SSD, take a look at the SSD page

Reply:
I left them unaligned because that's how they come out of /proc/mounts and it's not necessary to line them up. I wouldn't be opposed to formatting them here and leaving a note that it's not necessary to do so.

Reply:
That would be great!